### ISSUE TITLE
VSCode extension doesn't highlight variant group classes

### ISSUE THREAD

**Issue by itpropro** [unocss/unocss#1462]
If you are using variant groups, the first and (often) the last class in a variant group doesn't get parsed/detected by the vscode extensions.

![image](https://user-images.githubusercontent.com/15030951/186424414-1c5230b8-804c-42ef-a956-5626d3f9b152.png)

![image](https://user-images.githubusercontent.com/15030951/186424457-fbaa1854-5b27-4f66-8a86-a3296499d176.png)

Here is another example to test:

`before:(scale-0 transition ease-in-out duration-100 content-empty h-3 w-3 rounded-50% duration-100 bg-blue-500)`

**Comment by antfu** [comment]
Yes, it's not yet supported. PR welcome if anyone want to work on it.

**Comment by itpropro** [comment]
I would love to create a PR for that, where can I find some documentation on the vscode extension and which shared libraries/unocss imports it uses and how to debug that besides the normal MS documentation @antfu ?

**Comment by itpropro** [comment]
Thanks for the hint @antfu, I never worked with vscode extensions before. UnoCSS didn't compile in Windows (at least on my clean Win 11 node 16 VM), so I had to switch to WSL, but everything seems to work there. I have created PR #1468 that adds highlighting for the classes inside of the variant group. I hope, I find some time the next days to take a look at how to fix highlighting for the variant itself and maybe the annotation to include the variant.
